tests/common_suites.py

```diff
@@ -37,6 +38,7 @@
     MOCKDUO_NOVERIFY,
     MOCKDUO_PROMPTS_1,
```

Comment:
Completely unrelated to your change but why does GitHub render this in orange

Reply:
github works in mysterious (and colorful) ways